app/models/organisation.rb

```diff
@@ -1,15 +1,14 @@
 class Organisation < ActiveRecord::Base
+  include FuzzySearch
+
   belongs_to :organisation_type
   belongs_to :country
   belongs_to :district
   has_many :organisations_users
   has_many :orders
   has_many :users, through: :organisations_users
 
-  def self.search(search_text)
-    where("name_en ILIKE :search_text OR name_zh_tw ILIKE :search_text",
-      search_text: "%#{search_text}%")
-  end
+  configure_search props: [:name_en, :name_zh_tw], tolerance: 0.1
```

Comment:
@patrixr , use %i for an array of symbols. (Code Climate)

Reply:
@abhinav098 I'm afraid I disagree with this practice. 
If a good explanation is provided for it then I'll be happy to change it. But so far I don't understand why it is of any value